### PR TITLE
PS-8348 Release notes and release note tasks for 5.7.39-42

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   doc/source/_res/rn/5.7.39-highlights.txt
new file:   doc/source/_res/rn/deprecated-5.7.39.txt
new file:   doc/source/_res/rn/ps-mysql-blurb.txt
modified:   doc/source/conf.py
new file:   doc/source/release-notes/Percona-Server-5.7.39-42.rst
modified:   doc/source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/_res/rn/5.7.39-highlights.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Release Highlights
+=================================================
+
+Improvements and bug fixes provided by Oracle for *MySQL* 5.7.39 and included in Percona Server for MySQL are the following:
+
+* MySQL removes a 4GB tablespace file size limit on Widows 32-bit systems. The limit was set because of an incorrect calculation performed while extending the tablespace.
```
Comment (george-lorch) [inline doc/source/_res/rn/5.7.39-highlights.txt]: s/Widows/Windows

**Review by george-lorch — APPROVED** [review]

**Review by kaushikpuneet07 — APPROVED** [review]
LGTM

File: doc/source/_res/rn/5.7.39-highlights.txt (hunk 1 above)
Comment (alina-derkach-oaza) [reply doc/source/_res/rn/5.7.39-highlights.txt]: Thank you! Fixed.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-5.7.38-41.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   :local:
+
+.. include:: ../_res/rn/5.7.38-highlights.txt
+
+.. include:: ../_res/rn/deprecated-5.7.38.txt
+
+Bugs Fixed
+=================================================
+
+* :jirabug:`PS-6029`: Data masking gen_rnd_us_phone() function had a different format compared to MySQL upstream version.
```
Comment (patrickbirch) [inline doc/source/release-notes/Percona-Server-5.7.38-41.rst]: The data masking

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**